Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -783,7 +795,7 @@ class DropDownMenu
         event.stopPropagation();
         if (currentItemIndex >= 0)
         {
-          var itemAnchor = $(dropDownMenuItems[currentItemIndex]).children('a');
+          var itemAnchor = dropDownMenuItems[currentItemIndex].querySelector ('a')!;
```

Comment:
`.children` requires a `:scope >` prefix to only search for children. `querySelector` and `querySelectorAll` would otherwise go through all the descendants. See notes in the handbook to `.children("a")`

```suggestion
          var itemAnchor = dropDownMenuItems[currentItemIndex].querySelector (':scope > a')!;
```